src/app/calendar_rendering.js

```diff
@@ -0,0 +1,38 @@
+import { CALENDAR_TABLE_SELECTOR } from "./templates_and_constants";
+import { rowLayout } from "./templates_and_constants";
+import { dateCellLayout } from "./templates_and_constants";
+import { convertClassToSelector } from "./templates_and_constants";
+import { CALENDAR_TABLE_ROWS_CLASS } from "./templates_and_constants";
+import { CURRENT_MONTH_SELECTOR } from "./templates_and_constants";
+import { MONTHS } from "./templates_and_constants";
+import { DateArrangements } from "./date_arrangements";
+import { DateTypeSelection } from "./date_type_selection";
+
+export class CalendarRendering {
+    static calendarTable = document.querySelector(CALENDAR_TABLE_SELECTOR)
+    static currentMonth = document.querySelector(CURRENT_MONTH_SELECTOR)
+
+    static renderMonth (monthDataArray, year, month){ 
+        this.calendarTable.innerHTML = rowLayout(CALENDAR_TABLE_ROWS_CLASS)
+        let rows = document.querySelectorAll(convertClassToSelector(CALENDAR_TABLE_ROWS_CLASS))
+
+        monthDataArray.forEach((day, index) => {
+            if (new Date(year, month, day).getDay() === 1) {
```

Comment:
1. assign into variable for clarity instead using ctor in condition
2. new Date(year, month, day).getDay() === 1 could be instracted into separate function or method with descriptive name explaining what you are doing